libs/blocks/ost/ost.js

```diff
@@ -0,0 +1,41 @@
+import { loadScript, getConfig } from '../../utils/utils.js';
+
+const { miloLibs, codeRoot } = getConfig();
+const base = miloLibs || codeRoot;
+
+export default async function init() {
+  if (!window.ost) {
+    await loadScript(`${base}/deps/offer-selector-tool.js`);
+  }
+
+  const ostCountry = 'US';
+  const ostLanguage = 'en';
+  const ostEnvironment = 'STAGE';
+  const ostWcsApiKey = 'wcms-commerce-ims-ro-user-cc';
+  const ostAosApiKey = 'dexter-commerce-offers';
+  const ostAosAccessToken = sessionStorage.getItem('AOS_ACCESS_TOKEN') || localStorage.getItem('AOS_ACCESS_TOKEN');
+  const ostRootContainer = document.querySelector('.ost');
+  let url = new URL(window.location);
+  let searchParams = new URLSearchParams(url.search);
+  let osiSearch = searchParams.get("osi");
+  ostRootContainer.removeChild(ostRootContainer.firstElementChild);
+  window.ost.openOfferSelectorTool({
+    ostCountry,
+    ostLanguage,
+    ostEnvironment,
+    ostWcsApiKey,
+    ostAosApiKey,
+    ostAosAccessToken,
+    osiSearch
+  });
+  setTimeout(() => {
+    //prevents page flickering/scrolling
+    document.addEventListener("click", function() {
```

Comment:
Seems like there has to be a better alternative than this hack.  What happens without it?  Is the react app too large for the window?

Reply:
Without it the page scrolls to the bottom upon clicking on elements on the react app; seems to be caused by the react application itself. I'm aware this is a somewhat ugly "hack" so I have scheduled a meeting with the SME of this react code to find a better way to handle this.